test/node/regress-gh-329.js

```diff
@@ -0,0 +1,52 @@
+// Regression test for https://github.com/metarhia/jstp/pull/329
+
+'use strict';
+
+const test = require('tap');
+const jstp = require('../..');
+
+const app = new jstp.Application('app', {});
+let server;
+
+let inspectSent = false;
+
+test.beforeEach((done) => {
+  server = jstp.net.createServer([app]);
+
+  server.on('connect', (connection) => {
+    connection.on('incomingMessage', (message) => {
```

Comment:
I think it can be better to use server's `log` event here:
```js
server.on('log', (connection, event) => {
  if (event === 'inspect') {
    inspectSent = true;
  }
});
```